ml_backends/docker-compose.yml

```diff
@@ -0,0 +1,25 @@
+services:
+  example:
+    build:
+      context: ./example
+    volumes:
+      - ./example/:/app:z
+    ports:
+      - "2000:2000"
+  ngrok:
```

Comment:
This is an interesting approach, to have an ngrok service in the docker compose setup, but its not necessary. Ngrok is a tool you should have on your host machine. Launch the ml backend with `docker compose up`, then launch Ngrok it with `ngrok http 2000` in a separate shell tab. Then when you visit the full ngrok url, it will point to the ml backend.